omrmakefiles/rules.linux.mk

```diff
@@ -57,7 +57,11 @@ endif
 
 # Compile without exceptions
 ifeq (gcc,$(OMR_TOOLCHAIN))
-    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-rtti -fno-threadsafe-statics
+ifeq (1,$(OMR_RTTI))
+    GLOBAL_CXXFLAGS+=-fno-exceptions -fno-threadsafe-statics
+else
+	GLOBAL_CXXFLAGS+=-fno-exceptions -fno-rtti -fno-threadsafe-statics
+endif
 else
     GLOBAL_CXXFLAGS+=-qnortti -qnoeh
```

Comment:
-qnortti is also a flag for controlling RTTI when compiling with xlc.  Would you be able to make a similar change for this?